frontend_lib/test/component/CardPopup.spec.js

```diff
@@ -43,18 +46,19 @@ describe('<CardPopup />', () => {
       expect(wrapper.find(`.${props.customHeaderClass}.cardPopup__header`).prop('style').backgroundColor).to.equal(props.customColor)
     )
 
-    it(`should have its children ${children}   ${wrapper.find(`.cardPopup__body`).children()}`, () =>
-      expect(wrapper.find(`.cardPopup__body`).children().contains(children)).to.equal(true)
+    it(`should have its children`, () =>
+      expect(wrapper.find('.cardPopup__body').find(Children).length).equal(1)
     )
 
     it(`should be set hideCloseBtn to : ${props.hideCloseBtn}`, () => {
       expect(wrapper.find(`.cardPopup__close`)).to.have.lengthOf(1)
     })
   })
 
-  describe('Handlers test', () => {
-    it(`onClick handler should call the proper handler`, () =>
-      expect(wrapper.find(`.cardPopup__close`).prop('onClick')()).to.equal(props.onClose())
-    )
+  describe('Handlers', () => {
+    it(`onClick handler should call the proper handler`, () => {
```

Comment:
this test description isn't clear enough.
It should at least include the name of the props "onClose"